### PR TITLE
Feature/caes 862 redesign item

### PR DESCRIPTION
**Pull request by cu4xxi**
[WIP]
Redesign view/edit item

### CODE REVIEW

File: packages/components/ItemFields/Attachments.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    downloadFile(raw, name);
+  };
+
+  const handleClickDownloadAll = () => {
+    downloadAsZip(attachments);
+  };
+
+  const onClickRemove = () => {
+    console.log('onClickRemove');
```
Comment (beshkenadze) [inline packages/components/ItemFields/Attachments.js:79]: point

File: packages/components/ItemFields/Input.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  notification,
+  addonIcons,
+  className,
+}) => {
+  const [isEdit, setIsEdit] = useState(false);
+  const [value, setValue] = useState(propValue);
+  const [, copyToClipboard] = useCopyToClipboard();
+
+  const handleClickAcceptEdit = () => {
+    console.log('handleClickAcceptEdit: ');
```
Comment (beshkenadze) [inline packages/components/ItemFields/Input.js:91]: point

File: packages/components/ItemFields/ItemHeader.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  const handleClickCloseItem = () => {
+    dispatch(setWorkInProgressItem(null));
+  };
+
+  return (
+    <ColumnHeader>
+      <PathButton
+        color="white"
+        onClick={() => {
+          console.log('Change path');
```
Comment (beshkenadze) [inline packages/components/ItemFields/ItemHeader.js:49]: point

File: packages/components/ItemFields/ItemHeader.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      >
+        Personal
+        <Delimeter>|</Delimeter>
+        Passwords
+      </PathButton>
+      <ActionButton
+        icon="share"
+        color="white"
+        onClick={() => {
+          console.log('Share');
```
Comment (beshkenadze) [inline packages/components/ItemFields/ItemHeader.js:60]: point

**Comment by cu4xxi** [comment]
> @cu4xxi, if you want to use the plugs, then better to use notifications with the "feature is not implemented yet" text instead of the console.log command.

ok 👌  Got it

**Pull request merged**